e2e/_suites/fleet/fleet.go

```diff
@@ -565,51 +511,46 @@ func theAgentIsListedInFleetWithStatus(desiredStatus string, hostname string) er
 }
 
 func (fts *FleetTestSuite) theFileSystemAgentFolderIsEmpty() error {
-	agentInstaller := fts.getInstaller()
+	agentService := deploy.NewServiceRequest(common.ElasticAgentServiceName)
+	agentInstaller, _ := installer.Attach(fts.deployer, agentService, fts.InstallerType)
 
-	containerName := fts.getContainerName(agentInstaller)
+	pkgManifest, _ := agentInstaller.Inspect()
+	cmd := []string{
+		"ls", "-l", pkgManifest.WorkDir,
+	}
 
-	content, err := agentInstaller.ListElasticAgentWorkingDirContent(containerName)
```

Comment:
I think `ListElasticAgentWorkingDirContent` can be removed, as it's not used elsewhere